packages/repository/src/repositories/relation.repository.ts

```diff
@@ -26,7 +30,47 @@ export interface HasManyEntityCrudRepository<T extends Entity, ID> {
    * @param options Options for the operation
    * @returns A promise which resolves with the found target instance(s)
    */
-  find(filter?: Filter | undefined, options?: Options): Promise<T[]>;
+  find(filter?: Filter, options?: Options): Promise<T[]>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  update(entity: DataObject<T>, options?: Options): Promise<boolean>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  delete(entity: DataObject<T>, options?: Options): Promise<boolean>;
+  /**
+   * 
+   * @param entity 
+   * @param options 
+   */
+  save(entity: DataObject<T>, options?: Options): Promise<T | null>;
```

Comment:
What's the difference between `update` and `save`? I am proposing to omit the `save` method.

Reply:
@bajtos `save` means `patchOrCreate`

Reply:
While...I guess you would say that's not a good/clear name again lol, at least that's my thought, went into the code to see the difference.